build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -62,23 +85,29 @@ const currentFile = new Project({
 
 const previousTsConfigPath = `${previousBasePath}/tsconfig.json`;
 let previousFile: SourceFile;
-if (existsSync(previousTsConfigPath)) {
-	const project = new Project({
-		skipFileDependencyResolution: true,
-		tsConfigFilePath: previousTsConfigPath,
-	});
-	previousFile = project.getSourceFileOrThrow("index.ts");
+const project = new Project({
+	skipFileDependencyResolution: true,
+	tsConfigFilePath: existsSync(previousTsConfigPath) ? previousTsConfigPath : undefined,
```

Comment:
Just curious: why was this change needed?

Reply:
I felt it was a cleaner way to create the Project instance instead of reusing it in an else block, the way it used to be on line 72